api/pin.go

```diff
@@ -360,36 +364,37 @@ func (p *PinApi) removeDecryptionKeyFromChunkHash(ref []byte) []byte {
 	return ref
 }
 
-func (p *PinApi) GetNoOfChunks(rootHash string, isRaw bool, credentials string) uint64 {
+func (p *PinAPI) getNoOfChunks(rootHash string, isRaw bool, credentials string) uint64 {
 
 	noOfChunks := uint64(0)
 	walkerFunction := func(ref storage.Reference) error {
 		noOfChunks += 1
 		return nil
 	}
-	p.WalkChunksFromRootHash(rootHash, isRaw, credentials, walkerFunction)
+	p.walkChunksFromRootHash(rootHash, isRaw, credentials, walkerFunction)
 
 	return noOfChunks
 }
 
-// Used in testing
-func (p *PinApi) GetPinnedFiles() map[string]uint8 {
-	return p.db.GetPinFilesIndex()
-}
+//
+// Functions used in testing
+//
 
-func (p *PinApi) GetPinnedChunks() map[string]uint64 {
-	return p.db.GetPinnedChunks()
+// GetPinnedFiles is used in testing to get all the pinned file root hashes and weather it
```

Comment:
to get root hashes of all the pinned content including info whether they are raw files  or manifests

Reply:
Done